README.md

```diff
@@ -6,43 +6,127 @@ Primary developers:
 
 * _Your name here_
 
-# Development
+## Development
 
 ### First-time setup
 
-Check out the app with Boxen:
-
-    $ boxen app-prototype
-    $ cd ~/src/app-prototype
-
 To install the required gems & prepare the database:
 
-    $ bin/setup
+```
+bin/setup
+```
 
 Run `rake db:sample_data` to load a small set of data for development. See
 `db/sample_data.rb` for details.
 
 ### Running the Application Locally
 
-    $ foreman start
-    $ open http://localhost:3000
+```
+foreman start -f Procfile.dev
+open http://localhost:3000
+```
 
 ### Running the Specs
 
 To run all ruby and javascript specs:
 
-    $ rake
+```
+rake
+```
 
 Again, with coverage for the ruby specs:
 
-    $ rake spec:coverage
+```
+rake spec:coverage
+```
 
 ### Using Guard
 
 Guard is configured to run ruby and jasmine specs, and also listen for
 livereload connections. Growl is used for notifications.
 
-    $ guard
+```
+guard
+```
+
+### Assets
+
+Assets live outside the standard sprockets-based asset pipeline. We’re using webpack to (almost) entirely replace the asset pipeline, though it works in a similar way to the asset pipeline in practice: we’re not creating fancy bundles, just normal static JavaScript and CSS.
+
+Assets live in `./assets/targets`. Creating a folder there will create a named target that matches. This:
+
+```
+./assets/targets/public —> public.js, public.css
+```
+
+To add a new target, you can use the `asset_target` generator:
+
+```
+rails generate asset_target target-name
+```
+
+This would generate the following files:
+
+```
+./assets/targets
+  /target-name
```

Comment:
@timriley Dasherized is the general style for frontend stuff, so I think it’s probably right.